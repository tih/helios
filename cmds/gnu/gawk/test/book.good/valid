NF != 7 {
	printf("line %d, : %s\n",NR,$0) }
$1 ~ /[^A-Za-z0-9]/ {
	printf("line %d, nonalphanumeric user id: %s\n",NR,$0) }
$2 == "" {
	printf("line %d, : %s\n",NR,$0) }
BEGIN	{ FS = ":" }
