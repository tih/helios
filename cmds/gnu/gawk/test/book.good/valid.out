line 1, : root::0:0:Computer Science SUN4,002C,6501,6543:/:/bin/csh
line 3, : toor:0:0:Bourne-again Superuser,,,:/:
